src/content/en/updates/2018/07/pwacompat.md

```diff
@@ -63,18 +63,28 @@ async loaded bundle:
 ```html
 <link rel="manifest" href="manifest.webmanifest" />
 <!-- include PWACompat _after_ your manifest -->
-<script async src="https://cdn.jsdelivr.net/npm/pwacompat@2.0.9/pwacompat.min.js"
-    integrity="sha384-VcI6S+HIsE80FVM1jgbd6WDFhzKYA0PecD/LcIyMQpT4fMJdijBh0I7Iblaacawc"
-    crossorigin="anonymous"></script>
+<script async src="https://unpkg.com/pwacompat" crossorigin="anonymous"></script>
 ```
 
-PWACompat will fetch your manifest file and do the work needed for your user's browser, regardless of
-whether they're on a mobile device or desktop.
+PWACompat will fetch your manifest file and do the work needed for your user's browser, regardless
+of whether they're on a mobile device or desktop.
 
-## More details
+We still recommend adding at least one high-quality shortcut icon, for search indexing:
 
-For supported browsers, what does PWACompat actually do? As of July 2018, PWACompat will load your
-Web App Manifest and:
+```html
+<link rel="icon" type="image/png" href="icon-128.png" sizes="128x128" />
+```
+
+For more information, see the [best practices](https://github.com/GoogleChromeLabs/pwacompat).
+
+Note: Alternatively, consider loading the library after your main bundle with a dynamic `import()`
+stement. While it's great to have splash screens, they're not as important as the rest of your
+site's code!
+
```

Comment:
```suggestion

```html
<script type="module">
  if (
    /\b(iPad|iPhone|iPod)\b/.test(navigator.userAgent) &&
    !matchMedia("(display-mode: standalone)").matches
  ) {
    import('https://unpkg.com/pwacompat');
  }
</script>
```

Reply:
Give them something to copy & paste :-) I realized it can be loaded even lazier than based on UA sniffing, that's [what we do in Excalidraw](https://github.com/excalidraw/excalidraw/pull/1439).

I fear this code block might break out of the `Note:` block (check before merging).